app/lib/handlers/pen.dart

```diff
@@ -208,7 +208,7 @@ class PenHandler extends Handler<PenTool> with ColoredHandler {
           }
           break;
```

Comment:
you don't need break in switch case in dart, see: https://dart.dev/language/branches#switch-statements